src/Moryx/Logging/IModuleLogger.cs

```diff
@@ -64,13 +65,8 @@ public interface IModuleLogger : IEnumerable<IModuleLogger>, INamedChildContaine
         void LogException(LogLevel level, Exception ex, string message, params object[] formatParameters);
 
         /// <summary>
-        /// Active logging level
+        /// Define a target for all messages with a level of <see cref="LogLevel.Warning"/> or higher
         /// </summary>
-        LogLevel ActiveLevel { get; }
-
-        /// <summary>
-        /// Clone this instance to be used on a new object
-        /// </summary>
-        IModuleLogger Clone(Type targetType);
+        void SetNotificationTarget(Action<LogLevel, string, Exception> notificationTarget);
```

Comment:
This has been added and not implemented, was it necessary? Or it is because of inheritence?

Reply:
It should be implemented inside the Module